### PR TITLE
2021/august update

### PR DESCRIPTION
**Pull request by kate-hansen**

### CODE REVIEW

File: README.md
Comment (chriswhawkins) [inline README.md]:
```suggestion
2. [On-boarding](on-boarding.md)
```

File: benefits.md
Comment (chriswhawkins) [inline benefits.md]: Should we just move the FMLA policy to the handbook as a pr to close that referenced issue? (and hold off on these changes until it's merged?)

File: code-of-conduct.md
Comment (chriswhawkins) [inline code-of-conduct.md]:
```suggestion
Employment with Base Two is "At-Will", however please see our [exiting policy](exiting.md) for more information.
```

File: day-to-day.md
Comment (chriswhawkins) [inline day-to-day.md:176]: I think I'd be a proponent of just getting rid of this whole section and limiting the list to tools that we all have to use for the company to operate successfully as a business (i.e. Gsuite, Slack, Zoom, Harvest, Jira, Github, etc) and keeping personal/staff recommended apps to slack or some place more on-demand/current.

File: on-boarding.md
Comment (chriswhawkins) [inline on-boarding.md:32]: "Supervisor" feels a little off brand for us... It seems like we would want something a little less "managerial" sounding here and more like, "...everyone plays a part in helping a new team member get started... we always designate at least one teammate as a point of contact... yadda yadda... buddy?... friend?... helper?... facilitator?..." 🤷

File: benefits.md
Comment (sludemann) [reply benefits.md]: Just reviewed that document and it seems pretty unfinished at this point.  There's still a lot of unclear language there that given my upcoming need to use it I'd like to clarify before we bake it into the handbook.

Comment (chriswhawkins) [reply benefits.md]: Yeah, I guess my suggestion would be to link that wip document on the issue instead of including it in the handbook at this point.

Comment (sludemann) [reply benefits.md]: Ahh totally makes sense, misread  :ignore me:

File: business-development.md
Comment (tborres) [inline business-development.md]:
```suggestion
We're most successful with clients when we are not merely seen as a vendor trying to sell a solution but instead seen as a collaborator. Our goal is to make clients feel that we are hearing and understanding their problem fully, and brainstorming together to arrive at a possible solution. We think of this as trying to get on the same side of the table as the client. We want them to feel that we are good listeners and hearing any problems they might have - not just those that they believe are explicitly in our wheelhouse. The contracts and sales tend to flow naturally out of a relationship where a client first and foremost feels heard and understood.
```

File: on-boarding.md
Comment (tborres) [reply on-boarding.md:32]: I agree that "supervisor" feels off brand, but I also don't know a better word to use. If we do change it note that we use the term "supervisor" several other places in the handbook.

**Review by tborres — APPROVED** [review]

**Pull request merged**
